cafebabel/static/js/script.js

```diff
@@ -116,15 +116,11 @@ Array.from(document.querySelectorAll('h1.edit input')).forEach(inputh1 => {
 
 /* Auto-expand textarea when typing http://jsfiddle.net/hmelenok/WM6Gq/ */
 var textAreaAutoResize
-if (window.attachEvent) {
-  textAreaAutoResize = function(element, event, handler) {
-    element.attachEvent('on' + event, handler)
-  }
-} else {
-  textAreaAutoResize = function(element, event, handler) {
-    element.addEventListener(event, handler, false)
-  }
+
+textAreaAutoResize = (element, event, handler) => {
```

Comment:
this should be a `function textAreaAutoResize…` and remove the `var textAreaAutoResize`

Reply:
ok